tests/cypress/tests/Verify_URLs_for_policy.spec.js

```diff
@@ -0,0 +1,50 @@
+/* Copyright (c) 2020 Red Hat, Inc. */
+/* Copyright Contributors to the Open Cluster Management project */
+
+/// <reference types="cypress" />
+
+import { getConfigObject } from '../config'
+import { getDefaultSubstitutionRules } from '../common/views'
+const substitutionRules = getDefaultSubstitutionRules()
+var policyName = ''
```

Comment:
Just nitpicking... I think it is better to use `let` instead of `var`. Also, I would personally move both definitions into the `describe()` block.